src/core/bittorrent/torrenthandle.cpp

```diff
@@ -1055,12 +1055,12 @@ int TorrentHandle::downloadPayloadRate() const
     return m_nativeStatus.download_payload_rate;
 }
 
-int TorrentHandle::totalPayloadUpload() const
+qlonglong TorrentHandle::totalPayloadUpload() const
```

Comment:
When I see ```long long``` I read "I need the longest available integral type here", while ```int64``` obviously reads "I need 64 bits". Why to ignore this type of expressiveness?